packages/html-spec/src/listview/listview.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "../../schemas/component.schema.json",
+    "title": "Kendo Listview HTML Spec",
+    "component": "listview",
+    "selector": [
+        ".k-listview"
+    ],
+    "states": [
+        {
+            "name": "disabled",
+            "selector": [
+                ".k-disabled"
+            ]
+        },
+        {
+            "name": "focus",
+            "selector": [
+                ":focus",
+                ".k-focus"
+            ]
+        }
+    ],
+    "options": [
+        {
+            "name": "borders",
+            "values": [
+                {
+                    "name": "all",
+                    "selector": [
+                        ".k-listview-borders-all"
+                    ]
+                },
+                {
+                    "name": "horizontal",
+                    "selector": [
+                        ".k-listview-borders-horizontal"
+                    ]
+                },
+                {
+                    "name": "vertical",
+                    "selector": [
+                        ".k-listview-borders-vertical"
+                    ]
+                }
+            ]
+        }
+    ],
```

Comment:
I would suggest not to add the `borders` option to the spec, as this is only available in jQuery. The ThemeBuilder does not need to 'know' about this option.